packages/react-components/src/button/src/Button.css

```diff
@@ -182,162 +202,163 @@ a.o-ui-button {
 /* OUTLINE | ACTIVE */
 .o-ui-button-outline:active,
 .o-ui-button-outline.o-ui-button-active {
-    box-shadow: 0 0 0 1px var(--marine-500) inset;
-    color: var(--white);
-    background-color: var(--marine-500);
-}
-
-/* OUTLINE | ACTIVE | FOCUS */
-.o-ui-button-outline.o-ui-button-focus:active,
-.o-ui-button-outline.o-ui-button-focus.o-ui-button-active,
-.o-ui-button-outline:focus:active,
-.o-ui-button-outline:focus.o-ui-button-active {
-    box-shadow: 0 0 0 1px var(--marine-500) inset, var(--o-ui-focus-ring);
+    box-shadow: 0 0 0 1px var(--quaternary-border) inset;
+    color: var(--static-white);
+    background-color: var(--foreground-color-5);
 }
 
 /* OUTLINE | ACTIVE | DISABLED */
 .o-ui-button-outline[disabled]:active,
 .o-ui-button-outline[disabled].o-ui-button-active,
 .o-ui-button-outline.o-ui-button-disabled:active,
 .o-ui-button-outline.o-ui-button-disabled.o-ui-button-active {
-    box-shadow: 0 0 0 1px var(--marine-500) inset;
-    color: var(--marine-500);
-    background-color: var(--white);
+    box-shadow: 0 0 0 1px var(--quaternary-border) inset;
+    color: var(--secondary-text);
+    background-color: var(--foreground-color-1);
 }
 
 /* OUTLINE | PRIMARY */
 .o-ui-button-outline.o-ui-button-primary {
-    box-shadow: 0 0 0 1px var(--primary-500) inset;
-    color: var(--primary-500);
+    box-shadow: 0 0 0 1px var(--brand-border) inset;
+    color: var(--brand-text);
 }
 
 /* OUTLINE | PRIMARY | HOVER */
 .o-ui-button-outline.o-ui-button-primary:hover,
 .o-ui-button-outline.o-ui-button-primary.o-ui-button-hover {
-    box-shadow: 0 0 0 1px var(--primary-700) inset;
-    color: var(--primary-700);
-}
-
-/* OUTLINE | PRIMARY | FOCUS */
-.o-ui-button-outline.o-ui-button-primary:focus,
-.o-ui-button-outline.o-ui-button-primary.o-ui-button-focus {
-    box-shadow: 0 0 0 1px var(--primary-700) inset, var(--o-ui-focus-ring);
+    box-shadow: 0 0 0 1px var(--quaternary-border-hover) inset;
+    color: var(--brand-text-hover);
 }
 
 /* OUTLINE | PRIMARY | ACTIVE */
 .o-ui-button-outline.o-ui-button-primary:active,
 .o-ui-button-outline.o-ui-button-primary.o-ui-button-active {
-    box-shadow: 0 0 0 1px var(--primary-500) inset;
-    color: var(--white);
-    background-color: var(--primary-500);
+    box-shadow: 0 0 0 1px var(--brand-border) inset;
+    color: var(--static-white);
+    background-color: var(--foreground-color-brand);
 }
 
 /* OUTLINE | SECONDARY */
 .o-ui-button-outline.o-ui-button-secondary {
-    box-shadow: 0 0 0 1px var(--cloud-200) inset;
-    color: var(--cloud-500);
+    box-shadow: 0 0 0 1px var(--tertiary-border) inset;
```

Comment:
I don't think "tertiary", "quaternary" is the way to go. It reminds me too much of spacing chart "alpha", "bravo" mess. Might be even worst since it is prone to typing issue.